_clanky/neziskovky-automatizace-dashboardu.md

```diff
@@ -0,0 +1,97 @@
+
+*Tip: Je dobrou praxí změnit název JSON souboru hned po stažení. Nejlépe na nějaký lehce zapamatovatelný výraz, protože s názvem souboru budeme ještě pracovat.*
+
+{% include image.html url="../attachments/články/neziskovky-automatizace-dashboardu/obrázky/developer.webp" description="Volba typu ověřování. Je zde také možnost vygenerovat si JSON klíč znovu. Tuto funkci najdete v editaci svého nově vytvořeného servisního účtu v sekci Service Account/Servisní účty." %}
+
+##Jak na to
+
+Nejdříve si otevřete stažený JSON klíč v textovém editoru a zkopírujte si obsah položky client_email. Následně si vytvořte prázdný [Google Sheet][SHEETS] a zkontrolujte, zda-li jeho první stránka má název list1. Pokud ne, tak ji přejmenujte. Pak nasdílejte google sheet na úrovni editor s emailem, který jste právě zkopírovali z JSON souboru.
+
+{% include image.html url="../attachments/články/neziskovky-automatizace-dashboardu/obrázky/key.webp" description="JSON soubor s emailem pro sdílení. Sdílení emailu umožní Google sheets API ověřit, zda jste to skutečně vy, kdo chce data aktualizovat." %}
+
+Dalším krokem je pak [stažení python skriptu][SKRIPT], s kterým budeme dále pracovat v textovém editoru.
+
+{% include image.html url="../attachments/články/neziskovky-automatizace-dashboardu/obrázky/skript.webp" description="Skript po otevření v textovém editoru." %}
+
+Pro další práci jsou důležité 3 řádky - **9**, **19** a **21**. Začneme tedy řádkem číslo 9. Zde stačí vyměnit url adresu za adresu, na které je ke stažení CSV distribuce vaší datové sady. V případě že máte jiný formát než-li CSV, bude nutné skript upravit nahrazením výrazu **read_csv** také v řádku 10. Seznam možností a podrobný návod najdete například [zde][IMPORTOTHER]. V našem případě načítáme data z [NKOD][NKOD], konkrétne z datové sady [Neziskové organizace v ČR][NEZISKOVKY] a její distribuce v CSV. Formát CSV jsme zvolili z důvodu jednoduché manipulace a širokého využívání veřejností.
+
+{% include image.html url="../attachments/články/neziskovky-automatizace-dashboardu/obrázky/neziskovky.webp" description="Datová sada Neziskové organizace v ČR s distribucí CSV z které skript stahuje data." %}
+
+V řádku **19** nahradíte ještě cestu k JSON klíči tak, aby se směřovala do umístění JSON souboru na vašem pc. V řádku **21** pak sekvenci v zátvorce nahradíte vlastním ID google sheetu, se kterým chcete pracovat. Toto ID najdete v url řádku otevřeného google sheetu. 
```

Comment:
Zde i jinde bych místo "pc" psal "PC". I tak je to trochu archaické. Nešlo by prostě "počítač"?

Reply:
Krom toho někde už PC je, takže to není konzistentní